Essentials/src/config.yml

```diff
@@ -516,6 +516,12 @@ allow-bulk-buy-sell: true
 # This has no effect if the MOTD command or permission are disabled.
 delay-motd: 0
 
+# A list of commands that should have their complementary confirm commands disabled by default.
+# This is empty by default, for the latest list of valid commands see the latest source config.yml.
+default-disabled-confirm-commands:
```

Comment:
Why not just two config options, one for each command?

I'm not sure that having a list to disable just two confirmations is a more worthwhile approach than moving the whole confirmation system to `EssentialsCommand` and having a list to enable confirmations by default per command.

Reply:
It's always easier to have new features commented out to enable the features for later versions of the config.

Reply:
Shouldn't these be disabled by default though and we can add them to have it enabled?